_posts/newsletters/2024-07-22-newsletter.md

```diff
@@ -0,0 +1,322 @@
+* [Code for Thought: Digital Research Academy - A Grassroots Network for Trainers](https://codeforthought.buzzsprout.com/1326658/15369343-en-digital-research-academy-a-grassroots-network-for-trainers) by Peter Schmidt
+* [Code for Thought: The Turing Way Part 2: From Handbook to Community](https://codeforthought.buzzsprout.com/1326658/15355230-en-the-turing-way-part-2-from-handbook-to-community) by Peter Schmidt
+* [Code for Thought - CERN: Keeping the lights on with OpenStack and Kubernetes - Spyridon Trigazis](https://codeforthought.buzzsprout.com/1326658/15321543-en-cern-keeping-the-lights-on-with-openstack-and-kubernetes-spyridon-trigazis) by Peter Schmidt
+* [Funding Open Source Science Software](https://geoiac.notion.site/Funding-Open-Source-Science-Software-b11741a8c9a24a91b8a367629092f674) by  Georgia Iacovou
+
+  
+-----------------
+
+<a name="involved"></a>
+# **6. Get Involved**
+
+New guidance was recently released on how to create new [Affinity Groups](https://us-rse.org/wg/group-management/form-an-affinity-group) and [Working Groups](https://us-rse.org/wg/group-management/form-a-working-group). If you see an unmet need in the US-RSE community, we encourage you to create your own group!
+
+As always, we thank everyone who is already contributing for their help in shaping and growing the organization.
+
+[US-RSE Working Groups](https://us-rse.org/working-groups/):
+
+* [Code Review](https://us-rse.org/wg/code-review/)
+* [Diversity, Equity, and Inclusion](https://us-rse.org/wg/dei/)
+* [Education & Training](https://us-rse.org/wg/education_training/)
+* [Group Management](https://us-rse.org/wg/group-management/)
+* [Mentorship Program](https://us-rse.org/wg/mentorship-program/)
+* [RSE Empowerment in National Labs](https://us-rse.org/wg/rse-empowerment-national-labs/)
+* [Testing](https://us-rse.org/wg/testing/)
+* [User Experience](https://us-rse.org/wg/ux/)
+* [Website](https://us-rse.org/wg/website/)
+
+-----------------
+
+<a name="jobs"></a>
+# **7. Recent Job Postings**
+
+These opportunities were recently posted to the [RSE Opportunities page](https://us-rse.org/jobs/):
+
+1. [Software Engineer](https://careersearch.stanford.edu/jobs/software-engineer-26078): Stanford University, Stanford, CA or Remote Posted: Jul 19, 2024
+2. [Software Developer Beamline Controls](https://jobs.lbl.gov/jobs/software-developer-beamline-controls-6603): Berkeley Lab, Berkeley, California Posted: Jul 17, 2024
+3. [Research Software Engineer](https://jobs.bnl.gov/job/upton/research-software-engineer/3437/67378557872): Brookhaven National Laboratory, Upton, NY Posted: Jul 12, 2024  
+4. [Chief Technology Officer and Director for Software](https://recruiting2.ultipro.com/SPA1004AURA/JobBoard/57b96f30-6a4b-42cc-8f73-d417a17b54e9/OpportunityDetail?opportunityId=3f55028a-6d04-4f1d-84af-1d66d754a4f1): NOIRLab/AURA, Tucson, AZ, USA | La Serena, Chile | Hilo, HI, USA Posted: Jul 11, 2024
```

Comment:
As a side-effect of this entry containing pipe characters, "|", Jekyll winds up generating a table for it in the resulting HTML.  I recommend that you simpy change the | characters to the word "or" in this line.

Reply:
You could also use "/" as some other entries do if you prefer.